android/build.gradle

```diff
@@ -41,5 +41,5 @@ android {
 
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
-    implementation 'com.facebook.android:facebook-android-sdk:12.+'
```

Comment:
Please provide information on what version your project resolved to as `12.+` should resolve to the latest, which is `12.3` at the time of writing (https://github.com/facebook/facebook-android-sdk/releases)

Please run a clean build/install and provide information on what version your setup resolved.